helpers/helpers.js

```diff
@@ -5,7 +5,10 @@
 const micromark = require("./micromark.cjs");
 
 const { newLineRe } = require("./shared.js");
+const { nextLinesRe } = require("./shared.js");
```

Comment:
This and the previous line can be merged into one. I may push this to your branch myself.